### PR TITLE
 add french names of males and females and surnames

### PR DESCRIPTION
**Pull request by belkhoujaons**
add french names of males and females and surnames

### CODE REVIEW

File: src/names/fr.ts
Comment (ozdemirburak) [inline src/names/fr.ts:1]: This is the file to put the names.

**Comment by ozdemirburak** [comment]
Thanks for the pull request. 

Can you please revert `br.ts` and add French names to `src/names/fr.ts`. Also, can you change the case of names to title case, like "Abdalah" instead of "ABDALAH" etc.

**Comment by belkhoujaons** [comment]
I fixed the bugs there , sorry it was my bad, can you please re-review it ?

File: src/names/fr.ts
Comment (ozdemirburak) [reply src/names/fr.ts:1]: Please use this instead:

```js
const frenchNames = {
    0: [ // 2000 ~ 2022 - https://web.archive.org/web/20231016211828/https://www.insee.fr/fr/outil-interactif/3532172/xml/Top100prenoms.xml
        'Lucas', 'Hugo', 'Enzo', 'Louis', 'Nathan', 'Théo', 'Gabriel', 'Léo', 'Thomas', 'Jules', 
        'Arthur', 'Raphaël', 'Mathis', 'Tom', 'Maxime', 'Paul', 'Clément', 'Adam', 'Antoine', 'Ethan', 
        'Noah', 'Alexandre', 'Baptiste', 'Axel', 'Yanis', 'Alexis', 'Valentin', 'Mathéo', 'Sacha', 'Quentin', 
        'Mohamed', 'Nolan', 'Victor', 'Maxence', 'Gabin', 'Evan', 'Romain', 'Liam', 'Nicolas', 'Martin', 
        'Rayan', 'Samuel', 'Maël', 'Simon', 'Benjamin', 'Robin', 'Adrien', 'Pierre', 'Julien', 'Mattéo', 
    ],
    1: [ // 2000 ~ 2022 - https://web.archive.org/web/20231016211828/https://www.insee.fr/fr/outil-interactif/3532172/xml/Top100prenoms.xml
        'Emma', 'Léa', 'Manon', 'Chloé', 'Jade', 'Camille', 'Louise', 'Inès', 'Sarah', 'Clara', 
        'Lola', 'Lucie', 'Eva', 'Alice', 'Lina', 'Zoé', 'Juliette', 'Marie', 'Anaïs', 'Léna', 
        'Romane', 'Océane', 'Ambre', 'Mathilde', 'Jeanne', 'Lou', 'Lilou', 'Charlotte', 'Lisa', 'Anna', 
        'Laura', 'Julie', 'Pauline', 'Julia', 'Louna', 'Clémence', 'Margaux', 'Rose', 'Nina', 'Célia', 
        'Agathe', 'Justine', 'Alicia', 'Margot', 'Elisa', 'Léonie', 'Mila', 'Elise', 'Luna', 'Emilie',
    ]
};

export default frenchNames;
```

File: src/surnames/fr.ts
Comment (ozdemirburak) [inline src/surnames/fr.ts:1]: Please change this with the code below:

```js
const frenchSurnames = [ // https://web.archive.org/web/20231016213609/https://fr.wikipedia.org/wiki/Liste_des_noms_de_famille_les_plus_courants_en_France
    'Martin', 'Bernard', 'Thomas', 'Petit', 'Robert', 'Richard', 'Durand', 'Dubois', 'Moreau', 'Laurent', 
    'Simon', 'Michel', 'Lefebvre', 'Leroy', 'Roux', 'David', 'Bertrand', 'Morel', 'Fournier', 'Girard', 
    'Bonnet', 'Dupont', 'Lambert', 'Fontaine', 'Rousseau', 'Vincent', 'Muller', 'Lefevre', 'Faure', 'Andre', 
    'Mercier', 'Blanc', 'Guerin', 'Boyer', 'Garnier', 'Chevalier', 'Francois', 'Legrand', 'Gauthier', 'Garcia', 
    'Perrin', 'Robin', 'Clement', 'Morin', 'Nicolas', 'Henry', 'Roussel', 'Mathieu', 'Gautier', 'Masson', 
];

export default frenchSurnames;
```

**Review by ozdemirburak — CHANGES_REQUESTED** [review]

File: src/surnames/fr.ts
Comment (ozdemirburak) [reply src/surnames/fr.ts:1]: Also, please update the `names/fr.ts` with the code below:

```js
const frenchNames = {
    0: [ // 2000 ~ 2022 - https://web.archive.org/web/20231016211828/https://www.insee.fr/fr/outil-interactif/3532172/xml/Top100prenoms.xml
        'Lucas', 'Hugo', 'Enzo', 'Louis', 'Nathan', 'Théo', 'Gabriel', 'Léo', 'Thomas', 'Jules', 
        'Arthur', 'Raphaël', 'Mathis', 'Tom', 'Maxime', 'Paul', 'Clément', 'Adam', 'Antoine', 'Ethan', 
        'Noah', 'Alexandre', 'Baptiste', 'Axel', 'Yanis', 'Alexis', 'Valentin', 'Mathéo', 'Sacha', 'Quentin', 
        'Mohamed', 'Nolan', 'Victor', 'Maxence', 'Gabin', 'Evan', 'Romain', 'Liam', 'Nicolas', 'Martin', 
        'Rayan', 'Samuel', 'Maël', 'Simon', 'Benjamin', 'Robin', 'Adrien', 'Pierre', 'Julien', 'Mattéo', 
    ],
    1: [ // 2000 ~ 2022 - https://web.archive.org/web/20231016211828/https://www.insee.fr/fr/outil-interactif/3532172/xml/Top100prenoms.xml
        'Emma', 'Léa', 'Manon', 'Chloé', 'Jade', 'Camille', 'Louise', 'Inès', 'Sarah', 'Clara', 
        'Lola', 'Lucie', 'Eva', 'Alice', 'Lina', 'Zoé', 'Juliette', 'Marie', 'Anaïs', 'Léna', 
        'Romane', 'Océane', 'Ambre', 'Mathilde', 'Jeanne', 'Lou', 'Lilou', 'Charlotte', 'Lisa', 'Anna', 
        'Laura', 'Julie', 'Pauline', 'Julia', 'Louna', 'Clémence', 'Margaux', 'Rose', 'Nina', 'Célia', 
        'Agathe', 'Justine', 'Alicia', 'Margot', 'Elisa', 'Léonie', 'Mila', 'Elise', 'Luna', 'Emilie',
    ]
};

export default frenchNames;
```

**Comment by ozdemirburak** [comment]
I've added my comments. This is just an update to use more common names and surnames. Thanks a lot.

File: src/surnames/fr.ts
Comment (belkhoujaons) [reply src/surnames/fr.ts:1]: requested changes are submitted, please review it

**Comment by ozdemirburak** [comment]
Thanks.

**Pull request merged**